gh-126647: `Doc/using/configure.rst`: Add an entry for ``--enable-experimental-jit`` option (#126648)
python/cpython: Doc/using/configure.rst

   .. versionadded:: 3.13

> .. option:: --enable-experimental-jit
>    Enables :ref:`JIT compiler <whatsnew313-jit-compiler>`.
>    Possible values:
>       * ``no`` - build the interpreter without the JIT.
>       * ``yes`` - build the interpreter with the JIT.
>       * ``yes-off`` - build the interpreter with the JIT but disable it by default.
>       * ``interpreter`` - build the interpreter without the JIT, but with the tier 2 enabled interpreter.
> 
>    .. versionadded:: 3.13

Reviewer: ```suggestion
.. option:: --enable-experimental-jit=[no|yes|yes-off|interpreter]

   Indicate how to integrate the :ref:`JIT compiler <whatsnew313-jit-compiler>`.

   * ``no`` - build the interpreter without the JIT.
   * ``yes`` - build the interpreter with the JIT.
   * ``yes-off`` - build the interpreter with the JIT but disable it by default.
   * ``interpreter`` - build the interpreter without the JIT, but with the tier 2 enabled interpreter.

   .. versionadded:: 3.13
```

Reviewer: Is there a default value if you just pass `--enable-experimental-jit`?

Author: > Is there a default value if you just pass `--enable-experimental-jit`?

Yes, it's a `yes`.

Also, if you don't pass a `--enable-experimental-jit` it would be converted to `--enable-experimental-jit=no`

Reviewer: I think we should mention it then. Maybe by adding some "(default)" after the `yes` (either before the `-` or at the end of the option's description, up to you). Or even after the bullet list: 

```rst
By convention, ``--enable-experimental-jit`` is a shorthand for ``--enable-experimental-jit=yes``.
```

   .. versionadded:: 3.13

.. option:: --enable-experimental-jit=[no|yes|yes-off|interpreter]

   Indicate how to integrate the :ref:`JIT compiler <whatsnew313-jit-compiler>`.

   * ``no`` - build the interpreter without the JIT.
   * ``yes`` - build the interpreter with the JIT.
   * ``yes-off`` - build the interpreter with the JIT but disable it by default.
   * ``interpreter`` - build the interpreter without the JIT, but with the tier 2 enabled interpreter.

>    By convention, ``--enable-experimental-jit`` is a shorthand for ``--enable-experimental-jit=yes``.
>    
>    .. versionadded:: 3.13

Reviewer: ```suggestion
   By convention, ``--enable-experimental-jit`` is a shorthand for ``--enable-experimental-jit=yes``.

   .. versionadded:: 3.13
```

Ok the diff was NOT showing the whitespaces.
